ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -108,6 +108,12 @@
     reason="Tests are not running on IPI deployed cluster"
 )
 
+awsi3_deployment_required = pytest.mark.skipif(
```

Comment:
If this is skipping tests when it is configured for AWS I3 why is this marker named `awsi3_deployment_required`? Shouldn't it be something like `skipif_aws_i3`?

Reply:
Agreed, now changed to skipif_aws_i3.